src/modules/options/OptionsWidget_textEncoding.cpp

```diff
@@ -122,7 +122,7 @@ OptionsWidget_textEncoding::OptionsWidget_textEncoding(QWidget * parent)
 		szTmp.replace("kvirc_", "");
 		szTmp.replace(".mo", "");
 		m_pForcedLocaleCombo->insertItem(m_pForcedLocaleCombo->count(), szTmp);
-		if(szTmp == m_szLanguage)
+		if(QString::compare(szTmp, m_szLanguage, Qt::CaseInsensitive))
```

Comment:
`QString::compare` does not return a bool of true if equal.  If you want to compare for string equality, you need to do `QString::compare(...) == 0`.